settings.php

```diff
@@ -39,6 +39,21 @@
         get_string('maxbatchsize', 'logstore_xapi'),
         get_string('maxbatchsize_desc', 'logstore_xapi'), 30, PARAM_INT));
 
+    // Maximum batch size for failed records being replayed.
+    $settings->add(new admin_setting_configtext('logstore_xapi/maxbatchsizeforfailed',
+        get_string('maxbatchsizeforfailed', 'logstore_xapi'),
+        get_string('maxbatchsizeforfailed_desc', 'logstore_xapi'), 15, PARAM_INT));
+
```

Comment:
@mlynn-lp is this setting needed?

Reply:
This was in the requirements, but it could cause a disparity between processing failed logs and putting them back into the live log.